cetmix_tower_server/readme/newsfragments/4827.feature.md

```
@@ -0,0 +1 @@
+Select variables from dropdown list in the code fields
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)